src/meta/client/MetaClient.cpp

```diff
@@ -589,6 +588,21 @@ folly::Future<StatusOr<std::vector<SpaceIdName>>> MetaClient::listSpaces() {
     return future;
 }
 
+folly::Future<StatusOr<std::vector<std::string>>>
+MetaClient::runAdminJob(cpp2::AdminJobOp op, std::vector<std::string> paras) {
+    cpp2::AdminJobReq req;
+    req.set_op(op);
+    req.set_paras(paras);
+    folly::Promise<StatusOr<std::vector<std::string>>> promise;
+    auto future = promise.getFuture();
+    getResponse(std::move(req), [] (auto client, auto request) {
+                    return client->future_runAdminJob(request);
+                }, [this] (cpp2::AdminJobResp&& resp) -> decltype(auto) {
```

Comment:
this is not used.

Reply:
Didn't get the point, could you explain a bit more ?

Reply:
Change the code from` [this] (cpp2::AdminJobResp&& resp)` to `[] (cpp2::AdminJobResp&& resp)` is better?

Reply:
Ah, haha... got you . 'this' ...